web/docs/understand/formats/json.md

```diff
@@ -1,251 +1,152 @@
----
-description: Line-delimited JSON
----
+# json
 
-# JSON
+Reads and writes JSON.
 
-The `json` format in VAST represents [line-delimited
-JSON](https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON) objects
-according to a specified schema. That is, one line corresponds to one event. The
-object field names correspond to record field names.
+## Synopsis
 
-JSON can express only a subset [VAST's type
-system](../data-model/type-system.md). For example, VAST has
-first-class support for IP addresses but they are strings in JSON. To get the
-most out of your data and retain domain semantics, [define a schema for your
-JSON objects](../../use/import/README.md#provide-a-schema-for-unknown-types).
+Parser:
 
-## Parser
+```
+json [--selector=field[:prefix]]
+```
 
-Consider the this example JSON file `data.json`:
+Printer:
 
-```json
-{"ts":"2011-08-15T03:36:16.748830Z","uid":"CKmcUPexVMCAAkl6h","id.orig_h":"210.87.254.81","id.orig_p":3,"id.resp_h":"147.32.84.165","id.resp_p":1,"proto":"icmp","conn_state":"OTH","missed_bytes":0,"orig_pkts":1,"orig_ip_bytes":56,"resp_pkts":0,"resp_ip_bytes":0,"tunnel_parents":[]}
-{"ts":"2011-08-15T03:37:11.992151Z","uid":"CTluup1eVngpaS6e2i","id.orig_h":"147.32.84.165","id.orig_p":3923,"id.resp_h":"218.108.143.87","id.resp_p":22,"proto":"tcp","duration":3.006088,"orig_bytes":0,"resp_bytes":0,"conn_state":"S0","missed_bytes":0,"history":"S","orig_pkts":4,"orig_ip_bytes":192,"resp_pkts":0,"resp_ip_bytes":0,"tunnel_parents":[]}
-{"ts":"2011-08-15T03:37:12.593013Z","uid":"C4KKBn3pbBOEm8XWOk","id.orig_h":"147.32.84.165","id.orig_p":3924,"id.resp_h":"218.108.189.111","id.resp_p":22,"proto":"tcp","duration":3.005948,"orig_bytes":0,"resp_bytes":0,"conn_state":"S0","missed_bytes":0,"history":"S","orig_pkts":4,"orig_ip_bytes":192,"resp_pkts":0,"resp_ip_bytes":0,"tunnel_parents":[]}
+```
+json [--pretty] [--numeric-durations] 
```

Comment:
I don't think numeric durations should be handled here. Instead, that should be an operator of its own.

Reply:
So remove `--numeric-durations` from the docs? Trying to understand the action item. 🙂

Reply:
I would say so. The option does not exist now (and neither do the `--omit-*` options), but since we have no plans to migrate them we should remove them.